packages/dynamic_layouts/README.md

```diff
@@ -13,25 +13,135 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages).
 -->
 
-TODO: Put a short description of the package here that helps potential users
+TODO(DavBot02): Put a short description of the package here that helps potential users
 know whether this package might be useful for them.
 
 ## Features
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+TODO(snat-s): List what your package can do. Maybe include images, gifs, or videos.
+This package provides support for multi sized tiles and different layouts.
+Currently the layouts that are implemented in this package are `Staggered` and
+`Wrap`.
+
+You can have reversed and with horizontal Grids with this layouts.
+
+The following are some demos of how each of the grids look.
+
+A staggered grid demo:
+
+<video src="assets/staggered_grid_demo.mov">
```

Comment:
Some of these files are multiple MB big. That may be to large to be checked into this repository. Have you checked what other packages in this repository do to embed assets? Maybe we can move these assets to our assets repo: https://github.com/flutter/assets-for-api-docs

/cc @Piinks